godel_simple_gui/src/states/select_plans_state.cpp

```diff
@@ -36,26 +34,37 @@ void godel_simple_gui::WaitToSimulateState::onStart(BlendingWidget& gui)
 
   if (plan_names_.empty())
   {
+    gui.setLabelText("System Status:");
     Q_EMIT newStateAvailable(new ErrorState("No motion plans available. "
                                             "Please check surface selections and try again",
                                             new SurfaceSelectState()));
```

Comment:
No, it actually moves the user back to the select surfaces state and gives them a message to select a surface.